lib/tasks/journeys_report.rake

```diff
@@ -78,6 +78,7 @@ namespace :journeys do
           journeys: journeys,
         }
       end
+      print " #{moves.length} moves exported. "
```

Comment:
`puts` not `print`

Reply:
but I think you want `puts` here :-)

Reply:
I wanted to get this effect `Generating 2020-07-07-geoamey.json... 183 moves exported. done.`
Can I have puts not adding the newline, or should I not do that at all?    e.g potential issue with flushing the console buffer, or similar?

Reply:
I wanted that newline behaviour. Maybe I should add a ` $stdout.flush` as well?

Reply:
thats fine as it is - no need for `$stdout.flush`